forward-call/.env

```diff
@@ -1,6 +1,7 @@
-# description: The number you want to forward incoming calls to
+# description: Calls made to your Twilio number will get forwarded to this e164-formatted phone number
```

Comment:
tiny nit, but i'd like us to be consistent on `e164` vs `E.164`. My guess is you just copy pasted this here

Reply:
ill update everything to E.164 to make it more formal